tests/plugins/test_stop_autorebuild_if_disabled.py

```diff
@@ -118,17 +118,18 @@ def test_malformed_config(self, caplog):
         # flexmock(configparser.SafeConfigParser).should_receive('getboolean').\
         #     with_args('autorebuild', 'enabled').and_raise(configparser.Error)
         # assert this doesn't raise
```

Comment:
This comment is no longer correct. While here you could remove the commented-out code if that's no longer relevant.